src/foam/nanos/logger/LogMessage.js

```diff
@@ -30,7 +30,8 @@ Implement LastModifiedByAware to suppress 'modified by' comment in journal outpu
       name: 'severity',
       class: 'Enum',
       of: 'foam.nanos.logger.LogLevel',
-      toJSON: function(value) { return value && value.label; }
+      toJSON: function(value) { return value && value.label; },
+      javaJSONOutput: `((foam.nanos.logger.LogLevel) value).getLabel()`
```

Comment:
Will this break parsing of the journal entries back into the system?  I don't believe we parse labels into enums in java currently.

What's the motivation for this change?

Reply:
Implementing fromJSON as well would fix parsing.

Reply:
The motivation in this particular case to have have human readable log messages.  So severity shows as 'debug', 'error' rather than 0, or 5 for example. 
In this case we are explicitly using a WriteOnlyJDAO with no intent, in the short term, for reading it back.  But yes, should we need/want to fromJSON would be required.   Makes sense to add it now so it's complete if we later want an external log viewer, which would be the case if log messages are all sent to a NOC. 
@KateMengjunXia please implement fromJSON.